src/templates/docs/breadcrumb-stub.js

```diff
@@ -79,4 +72,17 @@ export default function BreadcrumbsStubPage(props) {
       </DocLayout>
     </LocaleProvider>
   );
-}
+};
+
+export default BreadcrumbsStubPage;
+
+export const Head = ({ pageContext: { title } }) => {
+  const pageMetaData = {
+    data: {
+      title,
+      description: ' ',
```

Comment:
Why did you use the space?

Reply:
This is black box for me:D I left it as it was before